src/components/RenderedDice/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { DiceColour } from "../../types";
+import styled from "styled-components";
+
+interface Props {
+  diceColour: DiceColour;
+  numOfDice: number;
+}
+
+interface ContainerProps {
+  diceColour: DiceColour;
+}
+
+const diceColourToRenderColour = new Map<DiceColour, string>([
```

Comment:
A map is overkill here, recommend a simple javascript object https://medium.com/front-end-weekly/es6-map-vs-object-what-and-when-b80621932373

i couldnt articulate why at first but reading that article helped me realise!